### ISSUE TITLE
Switch to postcss-preset-env

### ISSUE THREAD

**Issue by calebeby** [cloudfour/cloudfour.com-patterns#430]
`postcss-cssnext` has been deprecated: https://moox.io/blog/deprecating-cssnext/

[`postcss-preset-env`](https://github.com/csstools/postcss-preset-env) is a much better tool because it conditionally enables/disables plugins based on the features you want and the browsers you want to support. It is actively maintained and has a growing community.

[You can try it out here](https://preset-env.cssdb.org/playground)

**Comment by calebeby** [comment]
This will require some changes to the css files, because:

`cssnext` was not keeping up with changes in the standards of css features very well. For example:

- The css `color()` function was renamed to `color-mod()` in the standards process, but cssnext was not updated

`postcss-preset-env` ensures it always stays up to date with the latest standards for css features.

**Comment by tylersticka** [comment]
I argued against using cssnext because I didn't like the idea of there being this additional point of failure or maintenance that we were relying on. At the time, the argument for keeping it was that it would streamline maintenance, so I relented. But reading that blog post makes me think I may have been right.

`postcss-preset-env` looks like an improvement, especially given that it can consume `browserslist` (which is awesome). Though I'm still conflicted about relying on a middle-man package for plugin updates and such, I'd love to give it a try.
